starter-code/views/passport/login.ejs

```diff
@@ -0,0 +1,7 @@
+<h1>LOG IN HERE</h1>
+
+<form method="POST" action="/login">
+  <label>Username: <input type="text" name="username" placeholder="username"></label>
+  <label>Password: <input type="text" name="password" placeholder="password"></label>
+  <button type="submit" name="button">Signup</button>
```

Comment:
You don't need the `type` and `name` attributes because:

- The default value of `type` is `submit`
- `name` is only use for sending data associated with the button; this is rarely useful.

Reply:
Very good to know! 1 tip about forms a day, at this rate I'll be a form-master by the end of the week. Cheers